Sources/TokamakCore/Fiber/Fiber.swift

```diff
@@ -37,27 +37,23 @@ public extension FiberReconciler {
   /// After the entire tree has been traversed, the current and work in progress trees are swapped,
   /// making the updated tree the current one,
   /// and leaving the previous current tree available to apply future changes on.
-  final class Fiber: CustomDebugStringConvertible {
+  class Fiber {
```

Comment:
nit: Is `Fiber` supposed to be subclassed anywhere? If not, what's the purpose of `final` removal?